README.md

```diff
@@ -1,3 +1,11 @@
+# Frontera
+
+## Overview
+
+Frontera is a framework for building [crawl frontiers](http://nlp.stanford.edu/IR-book/html/htmledition/the-url-frontier-1.html). Designed with Scrapy in mind, but applicable to any web crawling project.
```

Comment:
I just googled for "framework implementation" it doesn't look like common phrase. You think it sounds english?